tests/integration/conftest.py

```diff
@@ -496,6 +539,8 @@ def pre_registered(shell, request):
     # system identity: <UUID>, we need to store just the system UUID
     original_registration_uuid = uuid_raw_output.split(":")[1].strip()
 
+    _remove_client_tools_repo_oracle(shell)
```

Comment:
shouldn't this be placed after the fixture's yield?

Reply:
This is intentional.
I want to remove the client repo before calling convert2rhel, so it's not backed up and/or used by the utility during any yum calls.

Reply:
Backing up by c2r would be a problem? Because by removing the repos we are setting the system in a way that it contain packages without the equivalent repository. I mean, is this something even the customer would do?

Reply:
Good point, we can try if it won't yield any issues.